BootloaderCorePkg/Library/AcpiInitLib/AcpiInitLib.c

```diff
@@ -380,6 +382,117 @@ UpdateMadt (
   return EFI_SUCCESS;
 }
 
+/**
+  This function updates FWST ACPI data structure based on ESRT table.
+
+  @param[in]  Current               Pointer to MADT table
+  @retval     EFI_SUCCESS           Patched the MADT
+              EFI_UNSUPPORTED       Sysinfo table not found.
+              EFI_OUT_OF_RESOURCES  If less MADT entries
+**/
+EFI_STATUS
+UpdateFwst (
+  IN UINT8   *Current
+  )
+{
+  UINT32        RsvdBase;
+  UINT32        EsrtTableSize;
+  UINT64        OemId;
+  EFI_STATUS    Status;
+  LOADER_GLOBAL_DATA              *LdrGlobal;
+  BOOT_LOADER_VERSION             *Version;
+  EFI_SYSTEM_RESOURCE_TABLE       *pEsrtTable;
```

Comment:
Please follow EDKII coding standard for variable names.   pEsrtTable should be EsrtTablePtr.   Similar for pEsrtEntry.